src/reflection/skrypnyk/PersonFactory.java

```diff
@@ -0,0 +1,36 @@
+package src.reflection.skrypnyk;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+
+public class PersonFactory {
+
+    public static void main(String[] args) throws ClassNotFoundException, IllegalAccessException, NoSuchFieldException, InvocationTargetException, NoSuchMethodException, InstantiationException {
+        printFields(new Person("Name", 82, "Lviv"));
+        System.out.println();
+        printFields(new Person("FirstName", 2, "Kyiv"));
+    }
+
+
+    public static void printFields(Person person) throws IllegalAccessException, NoSuchFieldException {
+        Class<? extends Person> personInfo = Person.class;
+        Field fieldName = personInfo.getDeclaredField("name");
+        fieldName.setAccessible(true);
+        Object name = (String) fieldName.get(person);
+        System.out.println("Name: " + name);
+
+        Field fieldAge = personInfo.getDeclaredField("age");
+        fieldAge.setAccessible(true);
+        Object age = fieldAge.get(person);
+        System.out.println("Age: " + age);
+
+        Field fieldAddress = personInfo.getDeclaredField("address");
+        fieldAddress.setAccessible(true);
+        Object address = (String) fieldAddress.get(person);
+        System.out.println("Address: " + address);
```

Comment:
Consider using for-loop, instead hardcoded retrieving fields.
Imagine you have 20+ fields. It will be hard to write the same code for each one.

*note: Field class has methods -- getName() and get()
use for-loop and this advice to change this code